src/types.ts

```diff
@@ -208,8 +211,9 @@ export interface RelationshipsQuery {
     min: number;
   };
   sourceExternalIds?: string[];
-  targetTypes?: string[];
+  targetTypes: boolean;
```

Comment:
As my comment above, variable name should be updated to reflect that its a `boolean`